Source/Bogus/DataSets/Lorem.cs

```diff
@@ -24,53 +26,61 @@ public string Word()
       }
 
       /// <summary>
-      /// Get some lorem words
+      /// Get a number of random lorem words
       /// </summary>
+      /// <param name="num">
+      /// The number of random lorem words to be gotten.
+      /// </param>
       public string[] Words(int num = 3)
       {
-         return Enumerable.Range(1, num).Select(f => Word()).ToArray(); // lol
+         return Enumerable.Range(1, num).Select(_ => Word()).ToArray();
       }
 
       /// <summary>
       /// Get a character letter.
       /// </summary>
-      /// <param name="num">Number of characters to return.</param>
+      /// <param name="num">The number of characters to return.</param>
       public string Letter(int num = 1)
       {
          if( num <= 0 )
             return string.Empty;
 
-         var w = Words(1)[0];
-         var c = Random.ArrayElement(w.ToArray());
-         return c + Letter(num - 1);
+         var words = Words(1)[0];
+         var characters = Random.ArrayElement(words.ToArray());
+         return characters + Letter(num - 1);
       }
 
       /// <summary>
       /// Get a random sentence of specific number of words. 
       /// </summary>
-      /// <param name="wordCount">Get a sentence with wordCount words. Defaults between 3 and 10</param>
-      /// <param name="range">Add anywhere between 0 to 'range' additional words to wordCount. Default is 0.</param>
+      /// <param name="wordCount">
+      /// Get a sentence with wordCount words. Defaults between 3 and 10
+      /// </param>
+      /// <param name="range">
+      /// Add anywhere between 0 to 'range' additional words to wordCount. Default is 0.
+      /// </param>
```

Comment:
Hi Bart. Thanks for the PR. I just have a few stylistic issues here. I prefer to have these XML docs `<param>`s to be as compact as possible. IE:

Preferred: `<param>fff</param>` 
vs.
```
<param>
fff
</param>
```

Mostly because:
* Sometimes I have parsers that dig into these XML docs and having to deal with new line terminated text between `<tag>`s can be an extra annoyance.
* Also, more compact XML the better, this way most of the screen real estate is taken up by code, no so much multi-line XML docs.

I'm okay with `<summary>` being multi-line, but no so much `<param>`.

Please, don't take it personally, I'm mostly nitpicking at this point; but if we're looking forward to other data sets I'd like to mention it before we get too far.

I always appreciate your PRs.

Reply:
It's always good to mention these kinds of things, I'll try to keep it in mind for future PRs. Don't worry about nitpicking, these PRs are already mostly my documentation OCD kicking in. You are the owner of the repository so I'll go by your rules.